### PR TITLE
Draft/refactoring

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
Freetime activity `pyblish-lite refactoring` is almost done.
- pyblish-lite controller was changed a lot
    - controller does not require hardcoded spliting into groups
    - it is expected that first plugin group are collectors
    - groups can be set via environment variables (optionally)
    - validators order can be set via environment variable (optionally)
- stop button is working more as pause button
- used default Qt objects where possible
- removed most of unused code (not all)
- terminal items have selectable text
- groups are colored by result
- many more...

**This is not final result but I think it is good start.
Please test and let me know!!!**

### CODE REVIEW

**Comment by mkolar** [comment]
It seems to be working quite well. Feels faster (no proper testing done though) and the selectable terminal items are awesome. Visually though there's stuff to be desired.

1. I'm not a big fan of the coloured category labels after they run. How about colouring the background (the dark strip) instead of the text?

2. Plus signs are cropped in the category labels

3. All the plugin labels are cropped on the bottom

4. can we make the category label BG rounded corners instead of cut-off?

image

File: pyblish_lite/util.py [hunk 1]
```diff
@@ -82,3 +86,224 @@ def collect_families_from_instances(instances, only_active=False):
...
+    # - this variable sets when validation button will hide and proecssing
+    #   of validation will end with ability to continue in process
+    default_validation_order = pyblish.api.ValidatorOrder + 0.5
+
+    # Group range can be set with environment "PYBLISH_GROUP_RANGE"
+    default_group_range = 1
+
+    # Group string can be set with environment "PYBLISH_GROUP_SETTING"
+    default_groups = {
+        pyblish.api.CollectorOrder + 0.5: "Collector",
```
Comment (mkolar) [inline pyblish_lite/util.py]: let's change these to verbs please.
```
Collect
Validate
Extract
Integrate
```

**Comment by mkolar** [comment]
another one. When scrolling in the terminal with logs that have longer outputs (formatted, dict for isntance), the scrolling jumps and its not possible to see all the data correctly.

pyblish_lite_new_bug

**Comment by mkolar** [comment]
The problem with scrolling is still happening. You can see it by opening up one of the longer logs (one that will reach the bottom edge and extend beyond it). When you then try to scroll down, it jumps somewhere much lower.

**Comment by iLLiCiTiT** [comment]
New issues found in Blender :: PySide2
image
EDITED:
Fixed

**Comment by iLLiCiTiT** [comment]
Standalone publisher can't toggle instances and plugins 
image

**Comment by iLLiCiTiT** [comment]
image
EDITED:
Fixed

**Comment by iLLiCiTiT** [comment]
Nuke 
image
EDITED:
Fixed

**Comment by iLLiCiTiT** [comment]
Hopefully most of issues solved

**Comment by iLLiCiTiT** [comment]
Artist view wrapping don't work as should.
image

EDITED:
Should be fixed now.

**Comment by iLLiCiTiT** [comment]
Maya 2020. Font is too thick to easily read text. 
image

EDITED:
Seems to be fixed with correct position of "Open Sans" font family in stylesheets.

**Comment by iLLiCiTiT** [comment]
Label on artist page is elliding but not right when text containg string without breaks
image

EDITED:
Fixed

**Comment by iLLiCiTiT** [comment]
Expendable widgets can't be shrinked if text in can't be wrapped
image

EDITED:
Fixed

**Comment by iLLiCiTiT** [comment]
Nuke on refresh
```
Traceback (most recent call last):
  File "C:\Users\jakub.trllo\Desktop\pype\pype-setup\repos\pyblish-lite\pyblish_lite\model.py", line 935, in on_rows_removed
    row_num, 0, len(remaining_rows) - 1
  File "C:\Users\jakub.trllo\Desktop\pype\pype-setup\repos\pyblish-lite\pyblish_lite\model.py", line 914, in _remove_rows
    self.mapping_to.pop(row)
```

Solved in https://github.com/pypeclub/pyblish-lite/pull/55

**Comment by iLLiCiTiT** [comment]
Records in perspective are not cleared on refresh.
image

Solved in https://github.com/pypeclub/pyblish-lite/pull/57

**Pull request merged**